global/global.schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://HPC-buildtest.github.io/schemas/global/outer.schema.json",
+  "$id": "https://HPC-buildtest.github.io/schemas/global/global.schema.json",
```

Comment:
This url will need to exist before we merge here, can you PR to the schemas repo to update the file there first?

Reply:
i don't follow, this PR is for the schemas repo. I guess you can merge HPC-buildtest/buildtest-framework#270 before merging this PR or vice-versa.

Reply:
oops my mistake - I was reviewing on my phone and fussed that up. This is perfecto! The one failing test is because the outer schema (previous name) is mentioned in the test, and that needs to be updated.

Reply:
okay this is now fixed in commit https://github.com/HPC-buildtest/schemas/pull/21/commits/5686eb646bdc1b60a272fe2334f73d77ae62463b i ran the test locally and it was passing.